README.md

```diff
@@ -53,7 +53,7 @@ See [Installing on Linux](install-linux.md) on setting up the project on Linux.
 2. In root of project, run `npm i --legacy-peer-deps` to install the React Native dependencies.
```

Comment:
Shouldn't we be running `npm install` without the legacy deps?

Reply:
Ok, sounds like this PR is still a WIP. Let me know when it's ready for review.

Reply:
@alexfigtree we'll tackle the legacy peer deps issue in a separate PR, fwiw.

Reply:
@alexfigtree @dmitrizagidulin this PR is ready for review (has been for a while although it was marked as draft, I marked it appropriately).